lib/core/src/Cardano/Wallet/Primitive/Types/RewardAccount.hs

```diff
@@ -0,0 +1,48 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- |
+-- Copyright: © 2018-2020 IOHK
+-- License: Apache-2.0
+--
+-- This module provides the 'RewardAccount' data type, used for staking
```

Comment:
`s/staking/delegation/`  Staking is a taboo word.

Can the description be more specific than "purposes"?

Reply:
Hi @rvl 

I've pushed a fix in c21aabab7e0ee2a318e056a5298800f6c99ca795.

I've removed the "purposes" bit.

Does this work for you? (Feel free to suggest alternative wordings.)